### PR TITLE
Programs for review 190318

### PR DESCRIPTION
**Pull request by naresuha**
Hi Santhosh Sir,
Good Morning!
Need your guidance for specific program with message mentioned as:
'Program with Super keyword Inheritance'.
When I run this program, firstname & lastname is not available in output. I used super key word to call or use constructor from super class 'person' in sub class 'employee'.

### CODE REVIEW

**Comment by naresuha** [comment]
Commit post review by Santosh Sir.

**Comment by naresuha** [comment]
Commit before submitting new request.

File: src/Ch7/DemoString.java [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+ * version 1.0
+ */
+
+package Ch7;
+
+public class DemoString {
+
+	public static void main(String[] args) {
+
+		String f_name = "Sanrus";
```
Comment (gitadminsanrus) [inline src/Ch7/DemoString.java:15]: f_name should be written as "fName"

File: src/Ch7/DemoString.java [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+		String l_name = "Training";
+		
+		String Institue = f_name +" " + l_name;
+
```
Comment (gitadminsanrus) [inline src/Ch7/DemoString.java:19]: variable names must follow camelCaseNotation

File: src/Ch7/EmployeeStatic.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+/**
+ * author Naresh P
+ * since 2018
+ * version 1.1
+ */
+package Ch7;
+
+class employee{
+	
+	int id;
```
Comment (gitadminsanrus) [inline src/Ch7/EmployeeStatic.java:11]: It is better to use 'private' for all the instance variables, and have getter and setter methods.

for example:
private int id;

public int getId() {
     return id;
}

public void setId(int id) {
    this.id = id;
}

File: src/Ch7/EmployeeStatic.java [hunk 4]
```diff
@@ -0,0 +1,41 @@
+//Demonstrate Non-Access Type 'Static'
+/**
+ * author Naresh P
+ * since 2018
+ * version 1.1
+ */
+package Ch7;
+
+class employee{
```
Comment (gitadminsanrus) [inline src/Ch7/EmployeeStatic.java:9]: put this class in a separate file "Employee.java"

File: src/Ch8/Access.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+/**
+ * author Naresh P
+ * since 2018
+ * version 1.1
+ */
+package Ch8;
+
+class A{
+	int i;
+	protected int j; // applied non-access modifier
```
Comment (gitadminsanrus) [inline src/Ch8/Access.java:11]: FYI: 'protected' keyword is an access modifier.

File: src/Ch8/FindAreas.java [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+public class FindAreas {
+	public static void main(String agrs[]){
+		
+		Figure f = new Figure(10,20);
+		triangle t = new triangle(10,20);
+		Rectangle r = new Rectangle(10,20);
+		
+		Figure Figuref;//Need to understand this!
+		
+		Figuref = r;
```
Comment (gitadminsanrus) [inline src/Ch8/FindAreas.java:59]: Is this class compiling? 'Figuref' must be written as 'Figure f'

**Review by gitadminsanrus — COMMENTED** [review]
Hi Naresh, the code logic looks good but please follow java naming conventions. Everything else looks good.

**Pull request merged**